drivers/usb/device/usb_dc_stm32.c

```diff
@@ -101,6 +101,16 @@ static const struct gpio_dt_spec ulpi_reset =
 #else
 #define USB_DC_STM32_FULL_SPEED             USB_OTG_SPEED_FULL
 #endif
+
+#if USB_OTG_HS_ULPI_PHY || DT_HAS_COMPAT_STATUS_OKAY(st_stm32u5_otghs_phy) || \
+USB_OTG_HS_EMB_PHYC
+static const struct stm32_pclken phy_pclken[] = STM32_DT_CLOCKS(DT_INST_PHANDLE(0, phys));
+#endif
+
+#if USB_OTG_HS_ULPI_PHY || DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs) || USB_OTG_HS_EMB_PHYC
```

Comment:
s/st_stm32_otghs/st_stm32u5_otghs_phy/?
If so, can define  `HAS_PHY_CLK` in the above `#ifdef/#endif` close.